service/service.go

```diff
@@ -124,11 +126,27 @@ func New(config Config) (*Service, error) {
 		}
 	}
 
+	// TODO: Remove once appcatalogentry CRs exist and have the name of the
+	// team responsible for the app.
+	//
+	//	https://github.com/giantswarm/roadmap/issues/26
```

Comment:
The issue is closed already. Maybe you are referring to others?

Reply:
Yes I know. @piontec when can you create the implementation issue?

I also don't really get why we can't use the same issue for spec and implementation.

Reply:
Wait, what do you mean by implementation issue :) ?

Reply:
Well the spec issue https://github.com/giantswarm/roadmap/issues/26 is closed.

But it isn't implemented yet. Where do we track that?